recipe/meta.yaml

```diff
@@ -108,41 +99,32 @@ outputs:
       build:
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-        - autoconf                # [not win]
-        - automake                # [not win]
-        - libtool                 # [not win]
         - cmake
         - libgomp                 # [linux]
-        - llvm-openmp             # [osx or win]
-        - autotools_clang_conda   # [win]
+        - llvm-openmp             # [osx]
       host:
         - libblas
         - liblapack
-      run:
-        - libblas
-        - liblapack
       run_constrained:
         - faiss-proc * {{ faiss_proc_type }}
 
     test:
       commands:
         # shared
-        - test -f $PREFIX/lib/libfaiss.so                           # [linux]
-        - test -f $PREFIX/lib/libfaiss.dylib                        # [osx]
-        - if not exist %PREFIX%\\Library\\bin\\libfaiss.dll exit 1  # [win]
+        - test -f $PREFIX/lib/libfaiss.so               # [linux]
+        - test -f $PREFIX/lib/libfaiss.dylib            # [osx]
+        - if not exist %LIBRARY_BIN%\faiss.dll exit 1   # [win]
+        # On windows, faiss.lib is an "import library";
+        # Deleting it breaks the faiss-builds
+        - if not exist %LIBRARY_LIB%\faiss.lib exit 1   # [win]
```

Comment:
This comment can probably be improved; I tried deleting `faiss.lib` during the `build-lib.bat`, but then the cmake-invocation in `build-pkg.bat` fails.